.github/workflows/deploy-release.yml

```
@@ -0,0 +1,238 @@
+name: Deploy a new release
+
+on:
+  push:
+    tags:
+      - 'v*.*.*'
+
+jobs:
+  cross-compile-ios:
```

```suggestion
  build-ios:
```
`cross` is confusing because below it says `use-cross: false`.
We can name the job the same way it is named in `build.yml`.

---

The naming is still accurate because we are still cross-compiling even if not using the tool `cross` to do so.

I changed the iOS runner to ubuntu and enabled `cross` to maintain consistency. I previously experimented with not using `cross` to try to solve a bug that had nothing to do with this.